lib/gateway/Shard.js

```diff
@@ -1108,13 +1108,25 @@ class Shard extends EventEmitter {
                     const reaction = packet.d.emoji.id ? `${packet.d.emoji.name}:${packet.d.emoji.id}` : packet.d.emoji.name;
                     if(message.reactions[reaction]) {
                         ++message.reactions[reaction].count;
+                        ++message.reactions[reaction].count_details[packet.d.burst ? "burst" : "normal"];
                         if(packet.d.user_id === this.client.user.id) {
-                            message.reactions[reaction].me = true;
+                            if(packet.d.burst) {
+                                message.reactions[reaction].me_burst = true;
+                            } else {
+                                message.reactions[reaction].me = true;
```

Comment:
@bsian03 Should I **always** make `.me` true, even when it's not a super reaction? And then obviously `me_burst` would also be true when it's a super reaction. Just for backwards compatability reasons really. What d'ya think?

Reply:
Yes, do this. Would be better to check against whether it's burst or not (or by checking type), if applicable